front/lib/production_checks/checks/scrub_deleted_core_document_versions.ts

```diff
@@ -202,6 +225,7 @@ async function scrubDocument({
 
   // New logic.
   const path = `${dataSource.project}/${dataSource.internal_id}/${documentIdHash}`;
+  const filename = `${timestamp}_${hash}.json`;
```

Comment:
Do we capture all deleted versions with this?

Reply:
timestamp and hash comes from the db, they uniquely identify one item that is marked as deleted . all files to delete are individually removed.

Reply:
Pretty sure than in core we use `created` 🤔

Reply:
yes , you're right. updated